src/quad_expr.jl

```diff
@@ -238,6 +245,43 @@ end
 function jump_function(model::AbstractModel, aff::MOI.ScalarQuadraticFunction)
     return QuadExpr(model, aff)
 end
+function jump_function(model::AbstractModel, f::MOI.VectorQuadraticFunction)
+    return QuadExpr[QuadExpr(model, f) for f in MOIU.eachscalar(f)]
+end
+
+"""
+    _fillvqf!(terms, offset::Int, oi::Int, quad::AbstractJuMPScalar)
+
+Fills the vectors terms at indices starting at `offset+1` with the quadratic
```

Comment:
This documentation can't be understood on its own. What is the vector terms? (A type would help.)